src/Workspaces/Core/Portable/Rename/IRemoteRenamerService.cs

```diff
@@ -74,64 +75,6 @@ public ValueTask<CodeCleanupOptions> GetOptionsAsync(RemoteServiceCallbackId cal
             => ((RemoteOptionsProvider<CodeCleanupOptions>)GetCallback(callbackId)).GetOptionsAsync(language, cancellationToken);
     }
 
-    [DataContract]
-    internal class SerializableSearchResult
```

Comment:
removed thsi concept entirely.  a SerializableRenameLocations points at a single SerializableSearchResult.  i've just inlined this for simplicity.